profiles/models.py

```diff
@@ -183,6 +183,7 @@ class Profile(TimestampedModel):
     highest_education = models.CharField(
         max_length=60, blank=True, default="", choices=HIGHEST_EDUCATION_CHOICES
     )
+    can_skip_application_steps = models.BooleanField(default=False)
```

Comment:
Can you add this to the admin class too? (`ProfileAdmin`)

Reply:
There is no such class in our project and if you could see the attached screenshots in the PR. This field is already showing in the profile admin section.

Reply:
Whoops! Sorry about that. Looks great